releases/2022.2/2022.2.md

```diff
@@ -0,0 +1,171 @@
+
+## Objectives
+
+This version had these major objectives:
+
+- Integrate a production-grade recommended document-oriented database for Napps.
+- Optimize `mef_eline` EVCs convergence to take place in a few seconds on average.
+- Enhance Kytos core to support event message prioritization and have higher performance when handling events in general.
+- Have a better infrastructure tools for application performance monitoring (APM) and logging.
+- New major features such as link liveness detection, interface loop detection, and new Web UI components.
+- Increase unit test coverage to at least 85% on `coloring`, `sdntrace`, and `sdntrace_cp`.
+- Leverage queue loggers to minimize latency.
+
+In order to achieve the objectives of this version, the following enhancements were refined and delivered:
+
+### 1. MongoDB core infrastructure and NApps integration
+
+MongoDB has been chosen as a default recommended document-oriented database based on the [EP026 blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP026.rst) requirements for NApps that need this type of storage. Kytos core has been integrated with `pymongo` driver library exposing a database client for NApps. Figure 1 illustrates the current high level integration between Kytos NApps and MongoDB with a replica set cluster.
+
+<div align="center">
+  <a href="assets/mongo_integration.png"><img src="assets/mongo_integration.png"></a>
+</div>
+<div align="center">
+  Figure 1 - Kytos NApps high level integration with MongoDB
+</div>
+
+On kytos-ng documentation, there are more technical information about the current [integration architecture](https://kytos-ng.github.io/napps/mongodb.html) with MongoDB and [general notes](https://kytos-ng.github.io/napps/mongodb_notes.html) for production deployment. On kytos [readme](https://github.com/kytos-ng/kytos#how-to-use-with-mongodb) page, there are more information about how to run and use MongoDB with Kytos.
+
+`flow_manager`, `mef_eline`, and `topology` have migrated from `storehouse` to MongoDB for higher performance, production and maintainability reasons. Also, since this new database infrastructure is officially supported, `storehouse` has been deprecated. If you were using any of these NApps in your network infrastructure, a migration script is also available in the `scripts/` folder of each NApp repository to migrate the stored data gracefully to leverage the new enhanced implementation and supported infrastructure.
+
+### 2. `mef_eline` EVCs convergence optimizations and QinQ enhancements
+
+TODO: Ask Italo to add and summarize the EVCs convergence gains.
+
+`mef_eline` has reintroduced setting the customer VLAN on the ingress when pushing the QinQ encapsulation since this action is correctly supported by OvS and NoviFlow switches. TODO: if it is worth mentioning any potential unsupported case that network operators should be aware of or any other rewording about this paragraph.
```

Comment:
@ajoaoff, if you could review this paragraph or any other information related to the TODO that you believe it's worth highlighting that'd be great